src/api/app/services/user_service/involved.rb

```diff
@@ -28,7 +38,7 @@ def involved_packages_and_projects
         involved_items.concat(pkg_or_prj_unfiltered(user: @user, klass: Package)) if consider_involved_packages?
         involved_items.concat(pkg_or_prj_unfiltered(user: @user, klass: Project)) if consider_involved_projects?
       end
-      involved_items.uniq
+      sort_involved_items(involved_items.uniq)
```

Comment:
@eduardoj you use `uniq` already inside the method

Reply:
True. Let's remove it from the method, then.

Reply:
Fixed. :+1: